pkg/lvm/lvm_util.go

```diff
@@ -254,6 +256,94 @@ func ResizeLVMVolume(vol *apis.LVMVolume, resizefs bool) error {
 	return err
 }
 
+// ResizeBTRFSVolume resizes the LVM volume as well as expand btrfs
+// Step1: Expanding LVM Volume (lvextend lvmvg -n <volume_name> -L size)
+// Step2: Expanding btrfs filesystem (btrfs filesystem resize max <mount_path>)
+func ResizeBTRFSVolume(vol *apis.LVMVolume, mountPath string) error {
+
+	desiredVolSize, err := strconv.ParseUint(vol.Spec.Capacity, 10, 64)
+	if err != nil {
+		return err
+	}
+
+	curVolSize, err := getVolumeSize(vol)
+	if err != nil {
+		return err
+	}
+
+	// Handle a case where LVM is expanded but not the filesystem
```

Comment:
Didn;t understand the comment and the following line of code. `desiredVolSize > curVolSize` means LVM is not expanded right?

Reply:
Yes, you are right `desiredVolSize > curVolSize` means LVM needs to be expanded... I need to update the comment as following `Trigger LVM resize only if it is required`